assignments/functionTypes.js

```diff
@@ -91,53 +207,157 @@ convertToString(21); // "21"
  * @param {number} y
  * @return {number} the result
  */
+    //Function declaration:
+    function calc(operation, x, y){
+      if(operation === "+"){
+        return x + y;
+      } else if (operation==="-"){
+        return x-y;
+      } else if (operation==="*"){
+        return x*y;
+      } else (operation==="/")
+        return x/y;
+    }
 
 /**
  * Returns true if `a` is greater than `b`.
  * @param {number} a
  * @param {number} b
  * @return {boolean} `a` is larger than `b`
- */
-
+ */ 
+//function declation:
+function larger(a,b){
+  return Boolean(a>b);
+}
+//function expression:
+let larger=function (a,b){
+  return Boolean(a>b);
+}
+//arrow function:
+let larger=(a,b)=> Boolean(a>b);
+//Arrow function with curley Bracket:
+let larger=(a,b) =>{
+  return Boolean(a,b);
+}
+ 
 /**
  * Returns true if `a` is less than `b`.
  * @param {number} a
  * @param {number} b
  * @return {boolean} `a` is smaller than `b`
  */
+//function declation:
+function smaller(a,b){
+  return Boolean(a<b);
+}
+//function expression:
+let smaller=function (a,b){
+  return Boolean(a<b);
+}
+//arrow function:
+let smaller=(a,b)=> Boolean(a<b);
+//Arrow function with curley Bracket:
+let smaller=(a,b) =>{
+  return Boolean(a<b);
+}
 
 /**
  * Returns true if `a` and `b` are equal.
  * @param {number} a
  * @param {number} b
  * @return {boolean} the numbers are equal
  */
+//function declation:
+function equals(a,b){
+  return Boolean(a=b);
```

Comment:
You are checking for equality `Boolean(a=b)`. Single `=` is assignment operator `==` is comparision. You need to use `==` to compare two values.